phys2bids/physio_obj.py

```diff
@@ -439,16 +439,16 @@ def check_trigger_amount(self, thr=None, num_timepoints_expected=0, tr=0):
         tr: float
             The Repetition Time of the fMRI data.
 
-        Notes
-        -----
-        Outcome:
-        self.thr: float
+        Returns
+        -------
+        :
+        thr: float
             Threshold used by the function to detect trigger points.
-        self.num_timepoints_found: int
+        num_timepoints_found: int
             Property of the `BlueprintInput` class.
             Contains the number of timepoints found
             with the automatic estimation.
-        self.timeseries:
+        shited_timeseries:
```

Comment:
Eeh... were you angry when writing this @vinferrer ? 😅

Reply:
Lol, just tired